irc/modes/modes.go

```diff
@@ -387,6 +387,18 @@ func (set *ModeSet) String() (result string) {
 	return buf.String()
 }
 
+// Strings returns the modes in this set.
+func (set *ModeSet) Strings() (result []string) {
```

Comment:
Nit: if this is only used by JWT I'd prefer if we moved it out of the modes package and into `channel.go` or even `handlers.go`.

Reply:
...yeah that totally makes sense, not sure why I added this in here when its use is really limited.